osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/ImportHighScoresCommand.cs

```diff
@@ -347,7 +366,7 @@ private ulong getMaxProcessable(Ruleset ruleset)
                 // when doing a single run, we need to make sure not to run into scores which are in the process queue (to avoid
                 // touching them while they are still being written).
                 using (var db = DatabaseAccess.GetConnection())
-                    return db.QuerySingle<ulong?>($"SELECT MIN(score_id) FROM score_process_queue WHERE is_deleted = 0 AND mode = {ruleset.RulesetInfo.OnlineID}") - 1 ?? ulong.MaxValue;
+                    return db.QuerySingle<ulong?>($"SELECT MIN(score_id) FROM score_process_queue WHERE is_deletion = 0 AND mode = {ruleset.RulesetInfo.OnlineID}") - 1 ?? ulong.MaxValue;
```

Comment:
My only doubt is that neither `is_deleted` nor `is_deletion` exist in my local database, so I'm unable to check which one is the correct if anything, but I suppose I'll just presume this is correct...

Reply:
Here's the new table structure. I think I posed in #infra as needing an update in the schema.

```sql
CREATE TABLE `score_process_queue` (
  `queue_id` int unsigned NOT NULL AUTO_INCREMENT,
  `score_id` bigint unsigned NOT NULL,
  `mode` tinyint unsigned NOT NULL,
  `start_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `update_time` timestamp NULL DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP,
  `status` tinyint(1) NOT NULL DEFAULT '0',
  `is_deletion` tinyint(1) NOT NULL DEFAULT '1' COMMENT 'marks not just deletion but basically anything that isn''t a new score',
  PRIMARY KEY (`queue_id`,`start_time`),
  KEY `status` (`status`),
  KEY `lookup_v3` (`mode`,`status`,`queue_id`),
  KEY `temp_pp_processor` (`score_id`)
) ENGINE=InnoDB AUTO_INCREMENT=2942130695 DEFAULT CHARSET=utf8mb3
```